fuzzers/baby/backtrace_baby_fuzzers/c_code_with_fork_executor/src/main.rs

```diff
@@ -46,7 +46,12 @@ pub fn main() {
         libafl::executors::ExitKind::Ok
     };
     // Create an observation channel using the signals map
-    let observer = unsafe { ConstMapObserver::<u8, 3>::from_mut_ptr("signals", map_ptr) };
+    let observer = unsafe {
+        ConstMapObserver::<u8, 3>::from_mut_ptr(
+            "signals",
+            ptr::NonNull::new(map_ptr).expect("map ptr is null."),
```

Comment:
I'd import `ptr::NonNull` and remove the `ptr` from here to make this slighly less yucky-looking

Reply:
ok

Reply:
i like `ptr::NonNull` because it makes it explicit what is non null, but i don't really have strong opinions there

Reply:
I mean I'd say if the `map_ptr` is `NonNull`, it's low-key obvious

Reply:
But up to you ultimately